audiomentations/augmentations/room_simulator.py

```diff
@@ -134,12 +155,12 @@ def __init__(
         self.min_source_z = min_source_z
         self.max_source_z = max_source_z
 
-        assert min_mic_radius <= max_mic_radius
+        assert min_mic_distance <= max_mic_distance
         assert min_mic_azimuth <= max_mic_azimuth
         assert min_mic_elevation <= max_mic_elevation
 
-        self.min_mic_radius = min_mic_radius
-        self.max_mic_radius = max_mic_radius
+        self.min_mic_radius = min_mic_distance
```

Comment:
Was this a conscious choice, or did you forget to also rename the left side of `=`?

```suggestion
        self.min_mic_distance = min_mic_distance
```